components/constellation/constellation.rs

```diff
@@ -2733,6 +2735,9 @@ where
         if let Err(e) = storage_ipc_receiver.recv() {
             warn!("Exit storage thread failed ({:?})", e);
         }
+        if let Err(e) = webgl_threads_receiver.recv() {
```

Comment:
I think this will deadlock if `self.webgl_threads == None`, so we should make this conditional to it being `Some`.

Reply:
should be fixed now, ready for review again.

Reply:
I don't think you pushed your changes.

Reply:
sorry my bad, pushed now.